### PR TITLE
FIX - Removed PostAuthorize.

### PR DESCRIPTION
**Pull request by MateuszSz93**

### CODE REVIEW

File: src/main/java/com/intive/patronage/toz/pet/PetsController.java [hunk 1]
```diff
@@ -71,11 +67,15 @@
...
     })
     @GetMapping(value = "/{id}")
-    @PostAuthorize("hasAnyAuthority('SA', 'TOZ') or " +
-            "(hasAnyAuthority('SA', 'TOZ', 'VOLUNTEER', 'ANONYMOUS') and " +
-            "(returnObject.name != null and returnObject.type != null and returnObject.sex != null))")
+    @PreAuthorize("hasAnyAuthority('SA', 'TOZ', 'VOLUNTEER', 'ANONYMOUS')")
     public PetView getPetById(@ApiParam(required = true) @PathVariable UUID id) {
-        return convertToView(petsService.findById(id));
+        Pet pet = petsService.findById(id);
+        if (UserInfoGetter.hasCurrentUserAdminRole() || (pet.getName() != null && pet
```
Comment (LukaszPiskadlo) [inline src/main/java/com/intive/patronage/toz/pet/PetsController.java]: Formatting

Comment (MateuszSz93) [inline src/main/java/com/intive/patronage/toz/pet/PetsController.java]: What kind of formatting do you have on mind? Do you mean to move `pet` to new line or `&& pet`? Or something other?

Comment (LukaszPiskadlo) [inline src/main/java/com/intive/patronage/toz/pet/PetsController.java]: Sorry for not being clear. I meant e.g moving `&& pet` or even `|| (pet.getName() != null && pet` to the next line.

File: src/main/java/com/intive/patronage/toz/comment/CommentController.java [hunk 2]
```diff
@@ -65,11 +65,14 @@
...
     @GetMapping(value = "/{id}")
     @ResponseStatus(HttpStatus.OK)
-    @PostAuthorize("hasAnyAuthority('SA', 'TOZ') or " +
-            "(hasAnyAuthority('SA', 'TOZ', 'VOLUNTEER') and returnObject.state == 'ACTIVE')")
+    @PreAuthorize("hasAnyAuthority('SA', 'TOZ', 'VOLUNTEER')")
     public CommentView getCommentById(@PathVariable UUID id) {
-        return ModelMapper.convertToView(commentService.findById(id), CommentView.class);
-
+        Comment comment = commentService.findById(id);
+        if (UserInfoGetter.hasCurrentUserAdminRole() || comment.getState().equals(Comment.State.ACTIVE)) {
```
Comment (mlotysz) [inline src/main/java/com/intive/patronage/toz/comment/CommentController.java]: I prefer it like that:
```
if (!(UserInfoGetter.hasCurrentUserAdminRole() || comment.getState().equals(Comment.State.ACTIVE))) {
 +     throw new NoPermissionException();           
}
 +        return ModelMapper.convertToView(comment, CommentView.class);

File: src/main/java/com/intive/patronage/toz/pet/PetsController.java [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package com.intive.patronage.toz.pet;
 
 import com.intive.patronage.toz.config.ApiUrl;
-import com.intive.patronage.toz.error.exception.InvalidImageFileException;
+import com.intive.patronage.toz.error.exception.NoPermissionException;
```
Comment (mlotysz) [inline src/main/java/com/intive/patronage/toz/pet/PetsController.java]: Why don't we create spring security authenticate aware method and not put logic there?

**Pull request merged**